deployment/terraform/cdn.tf

```diff
@@ -0,0 +1,62 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  origin {
+    domain_name = "${aws_route53_record.origin.fqdn}"
+    origin_id   = "originWebsite"
+
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "https-only"
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
+    }
+  }
+
+  enabled          = true
+  http_version     = "http2"
+  comment          = "GeoTrellis Website (${var.environment})"
+  retain_on_delete = true
+
+  price_class = "${var.cdn_price_class}"
+
+  # The trailing period at the end of the name is stripped off to comply with CloudFront's CNAME policy.
+  aliases = ["${replace(data.aws_route53_zone.external.name, "/.$/", "")}", "www.${replace(data.aws_route53_zone.external.name, "/.$/", "")}", "beta.${replace(data.aws_route53_zone.external.name, "/.$/", "")}"]
```

Comment:
The `beta.` subdomain is not needed here. That was in there before to prepare for an HTTPS version of the site.